### ISSUE TITLE
Pre meditation timer

### ISSUE THREAD

**Issue by jubalh** [nyxkn/meditation#6]
Thanks for your app!

It would be great to have a countdown/timer before the meditation starts. Some kind of preparation timer.
It would be useful in cases where I still need to prepare. Let's say I put my phone on a certain place in the room, hit start, then go to my cushion, make myself comfortable.
I would like to have the option to give myself 2 minutes for that before the actual session starts.

**Comment by nyxkn** [comment]
Hi! There is a "Start delay" option in the settings which does what you need. Currently it's only up to 1 minute. Do you think we should allow for more?
Also, if you hadn't seen that option, do you think maybe the name "start delay" wasn't very intuitive?

**Comment by jubalh** [comment]
> There is a "Start delay" option in the settings which does what you need. 

Indeed it is there! Thank you for pointing that out :)

> Currently it's only up to 1 minute. Do you think we should allow for more?

For me that's most likely enough

>  Also, if you hadn't seen that option, do you think maybe the name "start delay" wasn't very intuitive?

I'm not sure why I missed that setting. The `Interval duration` I found easily. Might have been just my mistake!
What I could imagine would make it slightly more obvious is to display something like "seconds" "minute" after the number to make it more obvious. But I think in this case I just overlooked it.
